tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -1136,6 +1136,39 @@ void test_constraint_preserving_bjorhus_corrections_dt_v_zero(
       DataVector(grid_size_each_dimension));
 }
 
+template <size_t VolumeDim>
+tnsr::aa<DataVector, VolumeDim, Frame::Inertial> wrapper_func_cp_v_minus(
+    const tnsr::a<DataVector, VolumeDim, Frame::Inertial>&
+        outgoing_null_one_form,
+    const tnsr::A<DataVector, VolumeDim, Frame::Inertial>& incoming_null_vector,
+    const tnsr::A<DataVector, VolumeDim, Frame::Inertial>& outgoing_null_vector,
+    const tnsr::aa<DataVector, VolumeDim, Frame::Inertial>& projection_ab,
+    const tnsr::Ab<DataVector, VolumeDim, Frame::Inertial>& projection_Ab,
+    const tnsr::AA<DataVector, VolumeDim, Frame::Inertial>& projection_AB,
+    const tnsr::aa<DataVector, VolumeDim, Frame::Inertial>&
+        char_projected_rhs_dt_v_minus,
+    const tnsr::a<DataVector, VolumeDim, Frame::Inertial>&
+        constraint_char_zero_plus,
+    const tnsr::a<DataVector, VolumeDim, Frame::Inertial>&
+        constraint_char_zero_minus,
+    const tnsr::a<DataVector, 3, Frame::Inertial>& char_speeds) {
+  std::array<DataVector, 4> char_speed_array{
+      char_speeds.get(0), char_speeds.get(1), char_speeds.get(2),
+      char_speeds.get(3)};
```

Comment:
We should really do non-owning datavector for each component, or maybe even change `constraint_preserving_bjorhus_corrections_dt_v_minus` to instead take the `tnsr::a`? This is doing a lot of copying for no reason.

Reply:
Okay, sounds fine. Thanks!